server/api/submission_handler.py

```diff
@@ -0,0 +1,35 @@
+from flask import jsonify, Blueprint, request
+from api import db
+from models import Contest, Submission, User
+from datetime import date
+submission_handler = Blueprint('submission_new_handler', __name__)
+
+
+@submission_handler.route('/contestImage/submission/<contest_id>/<user_id>', methods=['POST'])
```

Comment:
```suggestion
@submission_handler.route('/contestImage/submission/<contest_id>', methods=['POST'])
```

you don't need to pass in the user_id because it should get authenticated when the middleware is installed. or, it can get passed in through the data